lib/install/management/appliance.go

```diff
@@ -596,8 +596,14 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 	},
 	)
 
+	// fix up those parts of the config that depend on the final applianceVM folder name
 	conf.BootstrapImagePath = fmt.Sprintf("[%s] %s/%s", conf.ImageStores[0].Host, d.vmPathName, settings.BootstrapISO)
 
+	if len(conf.ImageStores[0].Path) == 0 {
```

Comment:
I suspect there is no doubt that ImageStores array has at least one element, correct? Is there any chance it does'n?